gramineproject/gsc
# Add support for password-protected signing key

## Description of the changes 

Now that core Gramine's `gramine-sgx-sign` tool supports the `--passphrase` argument, we don't need the `expect` tool.

Fixes #148 

Prerequisite: https://github.com/gramineproject/gramine/pull/1581

## How to test this PR? 
Use protected signing key for signing the GSC image for any workload. make sure to supply `passphrase` with signing command.

Examples:
Create protected signing key: `openssl genrsa -3 -aes128 -passout pass:test@123 -out enclave-key.pem 3072`

Sign GSC image: `./gsc sign-image <image_name> enclave-key.pem --passphrase test@123`


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gsc/173)
